### ISSUE TITLE
Installation fails with errors in notify_linux.go

### ISSUE THREAD

**Issue by mrusme** [jangxx/SchubsiGo#21]
Hey there, just tried installing this on Linux and got the following errors:

```
 △ ~ go install github.com/jangxx/SchubsiGo@latest
go: downloading github.com/jangxx/SchubsiGo v1.2.0
go: finding module for package github.com/GeertJohan/go.rice
go: finding module for package github.com/cratonica/trayhost
go: finding module for package github.com/shibukawa/configdir
go: finding module for package github.com/jangxx/go-poclient
go: finding module for package github.com/esiqveland/notify
go: finding module for package github.com/gorilla/mux
go: finding module for package github.com/skratchdot/open-golang/open
go: finding module for package github.com/godbus/dbus
go: downloading github.com/esiqveland/notify v0.11.0
go: downloading github.com/GeertJohan/go.rice v1.0.2
go: downloading github.com/jangxx/go-poclient v0.0.0-20200509114043-da642efce3a2
go: downloading github.com/cratonica/trayhost v0.0.0-20150112162955-98495206fd96
go: found github.com/GeertJohan/go.rice in github.com/GeertJohan/go.rice v1.0.2
go: found github.com/cratonica/trayhost in github.com/cratonica/trayhost v0.0.0-20150112162955-98495206fd96
go: found github.com/esiqveland/notify in github.com/esiqveland/notify v0.11.0
go: found github.com/godbus/dbus in github.com/godbus/dbus v4.1.0+incompatible
go: found github.com/gorilla/mux in github.com/gorilla/mux v1.8.0
go: found github.com/jangxx/go-poclient in github.com/jangxx/go-poclient v0.0.0-20200509114043-da642efce3a2
go: found github.com/shibukawa/configdir in github.com/shibukawa/configdir v0.0.0-20170330084843-e180dbdc8da0
go: found github.com/skratchdot/open-golang/open in github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
go: downloading github.com/godbus/dbus/v5 v5.0.3
# github.com/cratonica/trayhost
In file included from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c: In function ‘tray_icon_on_menu’:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:62:5: warning: ‘gtk_menu_popup’ is deprecated: Use '(gtk_menu_popup_at_widget, gtk_menu_popup_at_pointer, gtk_menu_popup_at_rect)' instead [-Wdeprecated-declarations]
   62 |     gtk_menu_popup(GTK_MENU(menu), NULL, NULL, NULL, NULL, 0, gtk_get_current_event_time());
      |     ^~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtklabel.h:34,
                 from /usr/include/gtk-3.0/gtk/gtkaccellabel.h:35,
                 from /usr/include/gtk-3.0/gtk/gtk.h:33,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:6,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
/usr/include/gtk-3.0/gtk/gtkmenu.h:138:12: note: declared here
  138 | void       gtk_menu_popup                 (GtkMenu             *menu,
      |            ^~~~~~~~~~~~~~
In file included from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c: In function ‘create_tray_icon’:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:73:5: warning: ‘gtk_status_icon_new_from_pixbuf’ is deprecated [-Wdeprecated-declarations]
   73 |     tray_icon = gtk_status_icon_new_from_pixbuf(pixbuf);
      |     ^~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtk.h:275,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:6,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
/usr/include/gtk-3.0/gtk/deprecated/gtkstatusicon.h:86:23: note: declared here
   86 | GtkStatusIcon        *gtk_status_icon_new_from_pixbuf    (GdkPixbuf          *pixbuf);
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:76:5: warning: ‘gtk_status_icon_set_tooltip_text’ is deprecated [-Wdeprecated-declarations]
   76 |     gtk_status_icon_set_tooltip_text(tray_icon, menu_title);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtk.h:275,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:6,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
/usr/include/gtk-3.0/gtk/deprecated/gtkstatusicon.h:137:23: note: declared here
  137 | void                  gtk_status_icon_set_tooltip_text   (GtkStatusIcon      *status_icon,
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
.go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:77:5: warning: ‘gtk_status_icon_set_visible’ is deprecated [-Wdeprecated-declarations]
   77 |     gtk_status_icon_set_visible(tray_icon, TRUE);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtk.h:275,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/linux/tray.c:6,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/platform/platform.h:10,
                 from .go/pkg/mod/github.com/cratonica/trayhost@v0.0.0-20150112162955-98495206fd96/trayhost.go:43:
/usr/include/gtk-3.0/gtk/deprecated/gtkstatusicon.h:151:23: note: declared here
  151 | void                  gtk_status_icon_set_visible        (GtkStatusIcon      *status_icon,
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~~~~
# github.com/jangxx/SchubsiGo
.go/pkg/mod/github.com/jangxx/!schubsi!go@v1.2.0/notify_linux.go:29:28: cannot use conn (type *"github.com/godbus/dbus".Conn) as type *"github.com/godbus/dbus/v5".Conn in argument to notify.New
.go/pkg/mod/github.com/jangxx/!schubsi!go@v1.2.0/notify_linux.go:35:21: notifier.ActionInvoked undefined (type notify.Notifier has no field or method ActionInvoked)
.go/pkg/mod/github.com/jangxx/!schubsi!go@v1.2.0/notify_linux.go:75:3: cannot use actions (type []string) as type []notify.Action in field value
.go/pkg/mod/github.com/jangxx/!schubsi!go@v1.2.0/notify_linux.go:76:3: cannot use map[string]"github.com/godbus/dbus".Variant{} (type map[string]"github.com/godbus/dbus".Variant) as type map[string]"github.com/godbus/dbus/v5".Variant in field value
.go/pkg/mod/github.com/jangxx/!schubsi!go@v1.2.0/notify_linux.go:77:3: cannot use int32(5000) (type int32) as type time.Duration in field value
```

**Comment by jangxx** [comment]
Looks like something has changed in the current version of `trayhost` and the proper package versioning wasn't released yet so it's always downloading the very latest version. I will take a look as soon as I get around to it. In the meantime you can use the prebuilt binary which was built before the breaking change was introduced.

**Comment by wokawoka** [comment]
I'm also experiencing the same issue
any update? it's the only single working pushover client I have found on linux
thank you so much!

**Comment by jangxx** [comment]
Okay wow, I completely forgot about this issue.

Dusted off my Go skills again and replaced the broken package with a working one. Can you test it to see if the current version works for you? If it does I'm gonna tag it as the proper 1.3.0 release.

**Comment by jangxx** [comment]
Since no one has responded for a full week now, I'm going to consider this issue as fixed in version 1.3.0.
